src/wp-includes/class-wp-image-editor.php

```diff
@@ -185,7 +186,7 @@ abstract public function stream( $mime_type = null );
 	 *
 	 * @since 3.5.0
 	 *
-	 * @return int[] {
```

Comment:
is `int[]` improper doc notation?

Reply:
@adamsilverstein No, but it's just not correct here. The array is an associative array with `width` and `height` keys, and `int[]` would only be correct if the array was an indexed array, just a list of integers.